src/screen/battle/visualize.rs

```diff
@@ -30,6 +34,11 @@ use crate::{
 
 const BLOOD_SPRITE_DURATION: i32 = 6;
 
+lazy_static! {
+    static ref SPRITES: HashMap<String, SpriteInfo> = ron::de::from_str(include_str!("../../../assets/sprites.ron"))
```

Comment:
`include_str!` loads file at compile time whereas `utils::deserialize_from_file` does it during runtime. Which one would you like to use?

Reply:
I definitely prefer not having any global variables, even for config-like stuff. So yeah, it should be loaded at runtime and stored as a new field in the `BattleView` struct I guess.

Reply:
FYI: I've started implementing what I've written above on top of your commits.